plutus-core/plutus-core/examples/PlutusCore/Examples/Builtins.hs

```diff
@@ -261,6 +262,18 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni ExtensionFun where
                 => afa -> afa)
             (\_ _ -> ExBudget 1 0)
 
+    toBuiltinMeaning _ver ScottToMetaUnit =
+        makeBuiltinMeaning
+            (mempty
```

Comment:
specifically, this is `(\_ -> ())`, right? a little confusing.

Reply:
In particular, it can't *do* anything with the argument, right?

Reply:
> specifically, this is `(\_ -> ())`, right? a little confusing.

Yeah, sorry, I guess I love to use `mempty` in this case too much, I'll fix it.

Reply:
> In particular, it can't do anything with the argument, right?

It can check if the argument is a constant and if it happens to be one, it can do all sorts of stuff with it (in particular, implement the behavior of any other builtin). In the branch that I DMed you some time ago it was also possible to apply the term that was passed to a builtin, but there's no such functionality on `master`. So on `master` it's only the constants stuff.